pkg/providers/v1/aws_loadbalancer.go

```diff
@@ -1512,6 +1512,7 @@ func (c *Cloud) ensureSSLNegotiationPolicy(ctx context.Context, loadBalancer *el
 		if !errors.As(err, &notFoundErr) {
 			return fmt.Errorf("error describing security policies on load balancer: %q", err)
 		}
+		return nil
```

Comment:
we shouldn't return here as PolicyNotFoundException would mean we have to create the policy